masschallenge/django-accelerator
# AC-5946: Add "Time in UTC" in company overview times for programs

**Changes introduced in AC-5946**:
- add 'Time in UTC' to overview deadline and start dates on the program model

**How to test**:
- ensure your migration history is linear i.e. the parent of the migration on this PR exists, if it doesn't exist, point it to the last run migration on your local
- run `make migrate`
- On accelerate, visit http://localhost:9191/admin/mc/program/39/change/, see that the overview deadline and start dates have the added `Time in UTC` text to alleviate confusion

- On development, we do not show the user that the time is in UTC.
